pkg/scheduler/util.go

```diff
@@ -19,17 +19,17 @@ package scheduler
 import (
 	"fmt"
 	"io/ioutil"
-	"strings"
-
-	"gopkg.in/yaml.v2"
 
 	"github.com/kubernetes-sigs/kube-batch/pkg/scheduler/conf"
 	"github.com/kubernetes-sigs/kube-batch/pkg/scheduler/framework"
 	"github.com/kubernetes-sigs/kube-batch/pkg/scheduler/plugins"
+	"gopkg.in/yaml.v2"
```

Comment:
keep the original format.

Reply:
what's original format?

Reply:
```
	"io/ioutil"

	"gopkg.in/yaml.v2"

	"github.com/kubernetes-sigs/kube-batch/pkg/scheduler/conf"
```

Reply:
don't we have tools to enforce this? oh right, it's reverted!

Reply:
will wait the tool is brought back to fix this. this shouldn't be fixed by hand.